pymodbus/client/serial.py

```diff
@@ -35,8 +35,8 @@ class AsyncModbusSerialClient(ModbusBaseClient):
     :param stopbits: Number of stop bits 1, 1.5, 2.
     :param handle_local_echo: Discard local echo from dongle.
     :param name: Set communication name, used in logging
-    :param reconnect_delay: Minimum delay in seconds.milliseconds before reconnecting.
-    :param reconnect_delay_max: Maximum delay in seconds.milliseconds before reconnecting.
+    :param reconnect_delay: Minimum delay in seconds before reconnecting.
```

Comment:
This does not say that fractions are accepted ! We have had issues with a very similar text, where users said it was not clear that milliseconds could be used.

Reply:
I do not mind a new wording, but it needs to state what is possible, not everybody make the connection between seconds and a float.

Reply:
Ups forgot this change should be done for all client types, since it is the same parameter.

Reply:
This change brought it in line with how the `timeout` parameter is documented. I've also never seen it being documented differently in another library, hence my confusion when I misread it the way it is currently written.

I've now added (float) after each parameter description, including the 'timeout' value where this also was not specified.